ui/readme.swg.md

```diff
@@ -184,6 +184,30 @@ Dr Tabelle "Suchbegriffe mit niedriger CR" isch no extra gfiltert für Begriffe
 
 Dr Chart "Entwicklung Suchen ohne Ergebnis" zeigt, wieviel Anteil von all d'Suche (net Begriffe) ohne Treffer an all Suchanfragen send.
 
+Älle Diagramme werda im Konteschd von noch mehr Data a'zeigt. Dodafür gibts folgende Möglichkeita:
```

Comment:
The Swebian readme currently only contains English text. Please paste the English version here or start translating the complete file :)

Reply:
Oh, ok. There are already other parts that are in Swäbian. And I think English is the least option. I will create another PR that replaces the English Part with the German one.

Reply:
That's strange as I recently asked Siggi to replace the mixed text with complete English content, and he did ...